### ISSUE TITLE
Can you provide Res2Net with Group Normalization pre-trained model

### ISSUE THREAD

**Issue by minan19605** [Res2Net/Res2Net-PretrainedModels#45]
Can you provide Res2Net with Group Normalization pre-trained model?

Hope can provide res2net50_v1b_26w_4s with GN

Thanks a lot

**Comment by gasvn** [comment]
Sorry, currently we don't have the gn pretrained model. We can train it for you if you really need it, but it will take some time as we have limited GPU resource.

**Comment by minan19605** [comment]
Is possible to tell me how to train from scratch. I will use my dataset, not imagenet.
Thank you.

**Comment by gasvn** [comment]
You can refer to this repo https://github.com/Res2Net/Res2Net-ImageNet-Training
Other tricks such as mixup and longer training epoch will further improve the performance.

**Comment by minan19605** [comment]
Thank you so much for your information.
